frontend/src/containers/Addgroup.js

```diff
@@ -0,0 +1,392 @@
+                    />
+                    <ImagePicker
+                      uploadOptionFirst
+                      className="image"
+                      dontLookupSocialMediaAvatars
+                      handleChange={image => appendGroupForm({image})}
+                      label="Select collective image"
+                      uploadImage={uploadImage}
+                      i18n={i18n}
+                      presets={['/static/images/repo.svg', '/static/images/code.svg', '/static/images/rocket.svg']}
+                    />
+                  </div>
+                </div>
+                <div className="sm-col-10 order-1 content-center">
+                  <div className="flex-auto">
+                    <div className="flex flex-column">
+                      <div className="Addgroup-label">Name</div>
+                      <Input
+                        name={'name'}
+                        value={name}
+                        onChange={(value) => appendGroupForm({name: value})}
+                        maxLength={50}
+                        placeholder="Rails Girls Atlanta"/>
+
+                      <div className="Addgroup-label">Slug</div>
+                      <Input
+                        name={'slug'}
+                        value={slug}
+                        onChange={(value) => appendGroupForm({slug: value})}
+                        maxLength={50}
+                        placeholder="railsgirlatl"/>
+
+                      <div className="Addgroup-label">Currency </div>
+                      <Select
+                        name={'currency'}
+                        value={currency || 'USD'}
+                        options={['USD', 'EUR', 'AUD', 'CAD', 'GBP', 'JPY', 'MXN']}
+                        handleChange={(value) => appendGroupForm({currency: value})}/>
+
+                      <div className="Addgroup-label">Website</div>
+                      <Input
+                        name={'website'}
+                        value={website}
+                        onChange={(value) => appendGroupForm({website: value})}
+                        maxLength={100}
+                        placeholder="http://railsgirlsatlanta.com"/>
+
+                      <div className="Addgroup-label">Video</div>
+                      <Input
+                        name={'video'}
+                        value={video}
+                        onChange={(value) => appendGroupForm({video: value})}
+                        maxLength={255}
+                        placeholder="http://railsgirlsatlanta.com"/>
+
+                      <div className="Addgroup-label">Help us on our mission to...</div>
+                      <CustomTextArea
+                        name={'mission'}
+                        value={mission}
+                        onChange={(value) => appendGroupForm({mission: value})}
+                        maxLength={100}
+                        placeholder="State the core mission of your collective"/>
```

Comment:
This placeholder is *asking for input*, while most of the others are *showing examples of input*.